contracts/VotingToManageEmissionFunds.sol

```diff
@@ -159,14 +165,16 @@ contract VotingToManageEmissionFunds is VotingTo {
         uint256 _emissionReleaseTime, // unix timestamp
         uint256 _emissionReleaseThreshold, // seconds
         uint256 _distributionThreshold, // seconds
-        address _emissionFunds
+        address _emissionFunds,
+        uint256 _minBallotDuration
```

Comment:
Since we add this param here, let's fix `migrations/2_deploy_contract.js` accordingly.